src/components/upload/FileUpload.js

```diff
@@ -1,36 +1,31 @@
-import styled from "styled-components";
 import FileDropZone from "./FileDropZone";
-import {useState} from "react";
 import Settings from "../common/settings";
-import Notifications from "../common/Notifications";
 import {Confirmation} from "../common/StyledComponents";
 import FetchHandler from "../common/FetchHandler";
+import {styled} from '@mui/system';
+import UploadIcon from '@mui/icons-material/Upload';
+import {useState} from "react";
+import Box from '@mui/material/Box';
+import * as React from 'react';
+import {Button,TextField} from '@mui/material';
+import Notifications from "../common/Notifications";
+import { useNavigate } from "react-router-dom";
 
+  const FileUploadRoot = styled('div')({
+    alignItems: "center",
+    boxShadow: '0 3px 5px 2px ',
 
-const FileUploadRoot = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-`;
-
-const Input = styled.input`
-  height: 2em;
-  border-radius: 4px;
-  border: none;
-  margin-right: 20px;
-`;
-
-const UploadButton = styled.button`
-  height: 2em;
-  width: 10em;
-`;
-
+    padding: '0 20px',
+  });
+  const Projectdecription= styled(TextField)({
+    marginTop:20 ,
+    required : true,
+    width : 450
```

Comment:
```suggestion
    marginTop: '20px',
    required: true,
    width: '450px'
```
Give pixel unit for consistency and remove whitespace before colon